projects/index.html

```diff
@@ -1,11 +1,11 @@
 ---
-title: Projects
-excerpt: "This is where we keep a collection of some of the projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
+title: Project Showcase
+excerpt: "This is where we keep a collection of some of the most interesting projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
```

Comment:
I have a feeling that saying "most interesting" would make some (most) people shy way from submitting projects to be included.

Maybe we can just call these "user-submitted projects"